### DISCUSSION TITLE
Can I install Quarto with the open-source version of rstudio-server?

### DISCUSSION THREAD

**Discussion by LawrenceGordon** [quarto-dev/quarto-cli#6203] [Q&A]
### Description

I already have an install of rstudio-server on my remote. Is it possible to install and use the Quarto .deb install with this version?

**Comment by cderv** [comment]
Quarto is independant of RStudio Server. So if you install Quarto and it is found on PATH, you'll be able to use at command line or through RStudio IDE. 

Did you already try this ?
